src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -28,7 +43,8 @@ import "./Common.sol";
 contract SecurityCouncilManager is
     Initializable,
     AccessControlUpgradeable,
-    ISecurityCouncilManager
+    ISecurityCouncilManager,
+    EIP712Upgradeable
```

Comment:
this broke storage

Reply:
pushed https://github.com/ArbitrumFoundation/governance/pull/316 on main

Reply:
good catch, added our own 712 so we dont need to inherit